flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/WindowTableFunctionTest.scala

```diff
@@ -257,4 +257,48 @@ class WindowTableFunctionTest extends TableTestBase {
     util.verifyRelPlan(sql)
   }
 
+  @Test
+  def testSessionTVF(): Unit = {
+    val sql =
+      """
+        |SELECT *
+        |FROM TABLE(SESSION(TABLE MyTable, DESCRIPTOR(rowtime), INTERVAL '15' MINUTE))
+        |""".stripMargin
+    util.verifyRelPlan(sql)
+  }
+
+  @Test
+  def testSessionTVFProctime(): Unit = {
+    val sql =
+      """
+        |SELECT *
+        |FROM TABLE(SESSION(TABLE MyTable, DESCRIPTOR(proctime), INTERVAL '15' MINUTE))
+        |""".stripMargin
+    util.verifyRelPlan(sql)
+  }
+
+  @Test
+  def testSessionTVFWithPartitionKeys(): Unit = {
+    val sql =
+      """
+        |SELECT *
+        |FROM TABLE(SESSION(TABLE MyTable PARTITION BY (b, a), DESCRIPTOR(rowtime), INTERVAL '15' MINUTE))
+        |""".stripMargin
+    util.verifyRelPlan(sql)
+  }
+
+  @Test
+  def testSessionTVFWithNamedParams(): Unit = {
```

Comment:
If the order of parameters is changed, it seems that verification cannot be passed now?
```sql
     SESSION(
         TIMECOL => DESCRIPTOR(rowtime),
         DATA => TABLE MyTable PARTITION BY (b, a),
         GAP => INTERVAL '15' MINUTE))
```

Do we also need to support this?

Reply:
I have double checked the Flip-145 and the user doc. I only find that `the DATA param must be the first`. That means we should allow the order changes about param DATA and GAP. But I think it's better to open a separate Jira for it. This pr focuses on the wrong PARAM name between `SIZE` in Calcite and `GAP` in Flink. WDYT?

Reply:
+1, we can handle it in a separate PR, it seems that other types of TVFs may also have similar issues.

Reply:
I also add a related test for it.